M365/MDO/EmailAuthChecker.ps1

```diff
@@ -0,0 +1,7180 @@
+            } elseif ($Issue -like "*Invalid subdomain policy*") {
+                return "Fix DMARC subdomain policy - Valid values are 'none', 'quarantine', or 'reject' - Microsoft DMARC Policies: $($URLs.DMARCSetup)"
+            } elseif ($Issue -like "*Low TTL*") {
+                # Only show TTL recommendations for Microsoft/Office 365 providers
+                if ($Providers -contains "Microsoft/Office 365") {
+                    # Extract domain and TTL from issue text
+                    if ($Issue -match "Low TTL for domain ([^\s]+) \((\d+) seconds\)") {
+                        $domain = $matches[1]
+                        $currentTTL = $matches[2]
+                        # Check if TTL is less than 3600
+                        if ([int]$currentTTL -lt 3600) {
+                            return "Increase DMARC record TTL for <strong>$domain</strong> from <strong>$currentTTL seconds</strong> to at least <strong>3600 seconds</strong> (1 hour) for better DNS caching and stability - Microsoft SPF Troubleshooting Guide: <a href='https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records' target='_blank'>https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records</a>"
+                        } else {
+                            return "Increase DMARC record TTL to at least 3600 seconds (1 hour) for better DNS caching and stability - Microsoft SPF Troubleshooting Guide: <a href='https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records' target='_blank'>https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records</a>"
+                        }
+                    } else {
+                        return "Increase DMARC record TTL to at least 3600 seconds (1 hour) for better DNS caching and stability - Microsoft SPF Troubleshooting Guide: <a href='https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records' target='_blank'>https://learn.microsoft.com/en-us/defender-office-365/email-authentication-spf-configure?view=o365-worldwide#troubleshooting-spf-txt-records</a>"
+                    }
+                } else {
+                    # Don't show TTL recommendations for non-Microsoft providers
+                    return ""
+                }
+            }
+        }
+        "DKIM" {
+            return "Fix DKIM syntax errors - Microsoft DKIM Configuration Guide: $($URLs.DKIMSetup)"
+        }
+        default {
+            return "Review email authentication configuration - Microsoft Documentation"
+        }
+    }
+}
+#endregion
```

Comment:
Please remove all the `#region` and `#endregion` from the script. Those aren't needed for every single function and bloats the size of the script. 

If you want a note of what the function is, use a normal comment or `#Description`